### ISSUE TITLE
panic: {"codespace":"sdk","code":24,"message":"gasprice must be greater than 100satoshi"}

### ISSUE THREAD

**Issue by deep2chain** [orientwalt/htdf#88]
```make new``` failed with panic
```
$ cd github.com/orientwalt/htdf
$ make new
$ hsd start
I[2021-08-19|22:13:47.454] Starting ABCI with Tendermint                module=main 
INFO[0000] /---------flag1----------/ true              
INFO[0000] currVersion=0                                
INFO[0000] LastBlockHeight=0                            
INFO[0000] Execute IncreaseTotalLoosenToken Successed increaseCoins 3000000000000000satoshi totalLoosenToken 3000000000000000satoshi 
INFO[0000] Execute IncreaseTotalLoosenToken Successed increaseCoins 3000000000000000satoshi totalLoosenToken 6000000000000000satoshi 
panic: {"codespace":"sdk","code":24,"message":"gasprice must be greater than 100satoshi"}

goroutine 1 [running]:
github.com/orientwalt/htdf/app/v0.(*ProtocolV0).initFromGenesisState(0xc000fe5800, 0x1709c18, 0xc00109f530, 0xc0001fc400, 0x13, 0xc00025ade0, 0xc0002acdc0, 0x2, 0x2, 0x0, ...)
        /media/gustav/Investigation/github.com/orientwalt/htdf/app/v0/protocol_v0.go:501 +0x1550
github.com/orientwalt/htdf/app/v0.(*ProtocolV0).InitChainer(0xc000fe5800, 0x1709c18, 0xc00109f530, 0xc0001fc400, 0x13, 0xc00025ade0, 0x73a7458, 0xed8b05e19, 0x0, 0xc0011e5a80, ...)
        /media/gustav/Investigation/github.com/orientwalt/htdf/app/v0/protocol_v0.go:524 +0x2c8
github.com/orientwalt/htdf/app.(*BaseApp).InitChain(0xc0010732c0, 0x73a7458, 0xed8b05e19, 0x0, 0xc0011e5a80, 0x9, 0xc0001fc2c0, 0x20ff088, 0x0, 0x0, ...)
        /media/gustav/Investigation/github.com/orientwalt/htdf/app/abci.go:91 +0x5b6
github.com/tendermint/tendermint/abci/client.(*localClient).InitChainSync(0xc000488240, 0x73a7458, 0xed8b05e19, 0x0, 0xc0011e5a80, 0x9, 0xc0001fc2c0, 0x20ff088, 0x0, 0x0, ...)
        /media/gustav/OS/go/pkg/mod/github.com/orientwalt/tendermint@v0.99.5/abci/client/local_client.go:223 +0x115
github.com/tendermint/tendermint/proxy.(*appConnConsensus).InitChainSync(0xc001064c20, 0x73a7458, 0xed8b05e19, 0x0, 0xc0011e5a80, 0x9, 0xc0001fc2c0, 0x20ff088, 0x0, 0x0, ...)
        /media/gustav/OS/go/pkg/mod/github.com/orientwalt/tendermint@v0.99.5/proxy/app_conn.go:65 +0x78
github.com/tendermint/tendermint/consensus.(*Handshaker).ReplayBlocks(0xc00026afb0, 0xa, 0x0, 0x140b7d2, 0x6, 0xc0011e5a80, 0x9, 0x65, 0x0, 0x0, ...)
        /media/gustav/OS/go/pkg/mod/github.com/orientwalt/tendermint@v0.99.5/consensus/replay.go:333 +0x6d5
github.com/tendermint/tendermint/consensus.(*Handshaker).Handshake(0xc00026afb0, 0x171a040, 0xc000022400, 0xc001073180, 0x203000, 0x203000)
        /media/gustav/OS/go/pkg/mod/github.com/orientwalt/tendermint@v0.99.5/consensus/replay.go:279 +0x4cc
github.com/tendermint/tendermint/node.doHandshake(0xc001073180, 0x17195d8, 0xc000284f38, 0xa, 0x0, 0x140b7d2, 0x6, 0xc0011e5a80, 0x9, 0x65, ...)
        /media/gustav/OS/go/pkg/mod/github.com/orientwalt/tendermint@v0.99.5/node/node.go:288 +0x1a7
github.com/tendermint/tendermint/node.NewNode(0xc001073180, 0x17050c8, 0xc00109b7c0, 0xc00007f160, 0x16eb0a0, 0xc0004a0ed0, 0xc00007f480, 0x159c680, 0xc00007f490, 0x170a430, ...)
        /media/gustav/OS/go/pkg/mod/github.com/orientwalt/tendermint@v0.99.5/node/node.go:611 +0x3c5
github.com/orientwalt/htdf/server.startInProcess(0xc00057d128, 0x159ddd8, 0x0, 0x0)
        /media/gustav/Investigation/github.com/orientwalt/htdf/server/start.go:102 +0x55d
github.com/orientwalt/htdf/server.StartCmd.func1(0xc001098580, 0x20ff088, 0x0, 0x0, 0x0, 0x0)
        /media/gustav/Investigation/github.com/orientwalt/htdf/server/start.go:54 +0xa5
github.com/spf13/cobra.(*Command).execute(0xc001098580, 0x20ff088, 0x0, 0x0, 0xc001098580, 0x20ff088)
        /media/gustav/OS/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:850 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001cd080, 0xc00109b400, 0x1435898, 0xc00100fef0)
        /media/gustav/OS/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /media/gustav/OS/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
github.com/tendermint/tendermint/libs/cli.Executor.Execute(0xc0001cd080, 0x159e218, 0x1410048, 0x9)
        /media/gustav/OS/go/pkg/mod/github.com/orientwalt/tendermint@v0.99.5/libs/cli/setup.go:89 +0x3c
main.main()
        /media/gustav/Investigation/github.com/orientwalt/htdf/cmd/hsd/main.go:81 +0x797
```

**Comment by deep2chain** [comment]
- https://github.com/orientwalt/htdf/blob/b312c96c48e9c2314756739280520e9d47d2501f/app/abci.go#L86~88
```go
	if 0 == app.deliverState.ctx.BlockHeight() {
		app.deliverState.ctx = app.deliverState.ctx.WithBlockHeight( req.GetInitialHeight() )
	}
```

**Comment by youngqqcn** [comment]
fixed `make new` at commit b22f3552ec58830c2dc05e2e4cadda42adcd4537

**Comment by deep2chain** [comment]
disable gas-checking for genesis-block at commit c97c9c9ed081683342f5a6fd679b1fc3a1380ee5
